components/settings.js

```diff
@@ -17,6 +17,15 @@ class Settings extends Component {
           value={ this.state.text }
         />
       </View>
+      <View>
+      <Text>'Plese enter the message you would like to send out in an emergeny.'
+      <TextInput
+      editable
+      multiline
+      style={ { height: 40, borderColor: 'gray' , borderWidth: 1 } }
+      onChangeText={ (text) => this.setState({ text }) }
+      value={ this.state.text }
+      />
```

Comment:
Suggest tabbing lines 23-27 one more time, and putting the `/>` at the end of line 27